apps/aecontract/src/aect_sophia.erl

```diff
@@ -22,23 +21,17 @@
 -type wrapped_code() :: #{ source_hash := aec_hash:hash()
                          , type_info   := [{binary(), binary(), binary(), binary()}]
                          , byte_code   := binary()
-                         , compiler_version := binary()
+                         , compiler_version => binary()
                          , contract_vsn := integer()
                          }.
 
 -export_type([ wrapped_code/0 ]).
 
-%% After hard fork Minerva, we accept ?SOPHIA_CONTRACT_VSN_2 in serialization
--define(SOPHIA_CONTRACT_VSN, ?SOPHIA_CONTRACT_VSN_2).
```

Comment:
The default was already pointing to VSN_2 - I think we were temporarily blind or something...

Reply:
I blame it to the hiolidays, but now it is at least very explicitely right.